File: Code/Hvs/Hvs.Entities/GameObjects/Spider.cs
using Hvs.Interfaces.Architecture;

namespace Hvs.Entities.GameObjects
{
	public class Spider : IEntity
	{
		public string DisplayName { get; set; }
		public int Level { get; set; }
		public int MaxHp { get; set; }
		public double DamageCoefficient { get; set; }
		public double SpeedCoefficient { get; set; }
		public double FrequencyStart { get; set; }
		public double FrequencyMax { get; set; }
		public double FrequencyPerLevel { get; set; }
		public string Description { get; set; }
		public long Id { get; set; }
	}
}

File: Code/Hvs/Hvs.Web/ClientApp/hvs/common/entities/Spider.ts
import { Injectable } from '@angular/core';

@Injectable()
export class Spider {
	public DisplayName: string;
	public Level: number;
	public MaxHp: number;
	public DamageCoefficient: number;
	public SpeedCoefficient: number;
	public FrequencyStart: number;
	public FrequencyMax: number;
	public FrequencyPerLevel: number;
	public Description: string;
	public Id: number;
}